docs/application/dotnet/samples/tizen-samples.md

```diff
@@ -267,385 +280,424 @@
+			<p>This sample application demonstrates how to work with app-common API in order to obtain application-specific information.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m2applicationcontrol.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/ApplicationControl" target="_blank"><strong>Application Control</strong></a></p>
-			<p>This Application Control sample application demonstrates how you can call the operation of other applications that provide operations such as pick, view, or compose.</p>
+			<p>This sample application demonstrates how you can call the operation of other applications that provide operations such as pick, view, or compose.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m3volume.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/VolumeController" target="_blank"><strong>Volume</strong></a></p>
-			<p>Volume application demonstrates how to retrieve and set device's volume levels.</p>
+			<p>This sample application demonstrates how to retrieve and set device's volume levels.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m4systeminfo.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/System-info" target="_blank"><strong>System-info</strong></a></p>
-			<p>The Sample System-info application demonstrate how to get information about system properties and capabilities of the device.</p>
+			<p>This sample application demonstrate how to get information about system properties and capabilities of the device.</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m5emailui.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/EmailUI" target="_blank"><strong>Email UI</strong></a></p>
-			<p>This Email UI sample application demonstrates how to configure the screen using a variety of objects.<br />
+			<p>This sample application demonstrates how to configure the screen using a variety of objects.<br />
 			(This sample is only a reference for constructing an application's UI layout. Some menus and UI components do not provide real application's functions.)</p>
 			</td>
 		</tr>
 		<tr>
 			<td><img alt="" height="267" src="media/m6galleryui.png" width="150" /></td>
 			<td>
 			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/UI/GalleryUI" target="_blank"><strong>Gallery UI</strong></a></p>
-			<p>This Gallery UI demonstrates how to configure the screen using a variety of objects.<br />
+			<p>This sample application demonstrates how to configure the screen using a variety of objects.<br />
 			(This sample is intended to guide how to make you application UI but not functionality.)</p>
```

Comment:
(This sample is intended to guide how to make your UI of an application but not the functionality of your application).

Reply:
fixed